packages/flutter/test/material/tabs_test.dart

```diff
@@ -3950,6 +3950,141 @@ void main() {
     expect(iconTheme.color, equals(selectedTabColor));
   });
 
+  testWidgets('TabBar colors labels correctly', (WidgetTester tester) async {
+    int color = 0x00000000;
+    MaterialStateColor buildMSC() {
+      final Color selectedColor = Color(color++);
```

Comment:
This isn't wrong but we typically don't write this kind of thing in the Flutter SDK. Creating a bunch of const color and text styles is only slightly more tedious (and a little more Flutter-conventional). And then buildMSC() can just have a pair of parameters.

Reply:
i just didn't want the next guy who may have to add a few more colors to test a few more cases to keep track of the numbers. will fix it anyway.